ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__uperf-npt-ocp14.yaml

```diff
@@ -0,0 +1,41 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-uperf-npt
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-uperf-npt-gcp
```

Comment:
You should add this test to the list of tests thats in this file instead of creating a new one. I think they should be identical: https://github.com/openshift/release/blob/b5d909ce791dc49320df1bf6af89d1fafb48c1e3/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml#L23

Reply:
I have fixed this issue.